data_handler.py

```diff
@@ -59,14 +58,63 @@ def _filter_conditions(inputs) -> List:
     def _filter_vectorized(inputs) -> pd.DataFrame:
         start_time = time.time()
         print(inputs)
-
-        conditions = DataHandler._filter_conditions(inputs)
+        conditions = DataFilterer._filter_conditions(inputs)
         filtered = np.where(reduce(lambda a, b: a & b, conditions))
-        data = DataHandler.df.iloc[filtered].to_dict("records")
+        data = LoadedData.loaded_data.iloc[filtered].to_dict("records")
         print("---Filtering took %s seconds ---" % (time.time() - start_time))
-
         return data
 
+    @staticmethod
+    def search_data(inputs) -> dict:
+        # Alternative solution for exception? Only thrown in one case but i want to avoid several if statements
+
+        filtered_data = DataFilterer._filter_vectorized(inputs)
+        search_result = {}
+        search_result["data"] = filtered_data
+        search_result[
+            "number patients"
+        ] = f"Antal patienter: {len(filtered_data)} / {LoadedData.number_patients}"  # refactor string when import data functionality is added
+        return search_result
+
+
+class LoadedData:
+    # Class stores loaded data and information about it, ex number of patients
+
+    COLUMNS = [
+        "Behandlingsnr",
+        "Anmälningstidpunkt",
+        "SistaOpTidpunkt",
+        "Opkategori_text",
+        "Prioritet_dagar",
+        "ASAklass",
+        "KravOperationstidMinuter",
+        "KravFörberedelsetidMinuter",
+        "KravtidEfterMinuter",
+        "De_PlaneradOpsal_FK",
+        "PlaneradStartOpsalTidpunkt",
+        "PatientÅlderVidOp",
+        "Veckodag",
+        "Starttimme",
+        "TotaltidStart",
+        "Vårdform_text",
+        "Statistikkod",
+        "OpkortText",
+        "dagar_till_kritisk",
+    ]  # Necessary because callbacks will try to search when program is built, key error exception will be thrown, this is a temp fix
+    loaded_data = pd.DataFrame(columns=COLUMNS)
+    number_patients = 0
+
+    @staticmethod
+    def load_data(filename, content):
```

Comment:
I'm wondering if we want to have a try:except: structure for handling this.
```python
try:
    ...
except:
    raise RuntimeError("Something bad happened")
```

Or do we wish to change the structure such that we don't expect the file to exist, and simply create a new one if none exists?